src/main/java/com/google/api/codegen/transformer/csharp/CSharpStandaloneSampleTransformer.java

```diff
@@ -29,13 +30,15 @@
 import com.google.api.codegen.util.csharp.CSharpAliasMode;
 import com.google.api.codegen.viewmodel.CallingForm;
 import com.google.api.codegen.viewmodel.ClientMethodType;
+import com.google.api.codegen.viewmodel.SampleEntryPointView;
 import com.google.common.collect.ImmutableMap;
 import java.util.function.Function;
 
 /** A transformer that generates C# standalone samples. */
 public class CSharpStandaloneSampleTransformer extends StaticLangGapicSamplesTransformer {
 
   private static final String STANDALONE_SAMPLE_TEMPLATE_FILENAME = "csharp/standalone_sample.snip";
+  private static final String PROGRAM_TEMPLATE_FILENAME = "csharp/gapic_samples_program.snip";
```

Comment:
Should we call this `..._runner` rather than `..._program`?

Reply:
same as below, `program` is commonly used as the name of an entry point class.